core/src/main/resources/hudson/tasks/Fingerprinter/FingerprintAction/index.jelly

```diff
@@ -65,11 +66,6 @@ THE SOFTWARE.
             <td data="${-f.timestamp.time}">
               ${f.timestampString} old
             </td>
-            <td>
-              <a href="${rootURL}/fingerprint/${f.hashString}/">
-                <l:icon class="icon-fingerprint icon-sm"/> ${%more details}
```

Comment:
do any localised properties need removing since this is no longer in use?

Reply:
This is actually just shifted a few lines above, so probably not needed. However I did notice xmlns:d="jelly:define" and xmlns:f="/lib/form"> is not needed even before this PR. Should I remove it?

Reply:
up to you, we normally leave them alone so as not to conflict with other PRs, but you're likely to be the only one touching fingerprints so it's safe to clean it up.

Reply:
@timja thanks! I removed it :)